build/Makefile

```diff
@@ -413,6 +414,12 @@ build-licenses:
 build-required-src-dist:
 	docker run --rm $(common_mounts) $(build_tag) $(mount_path)/build/build-required-src-dist.sh
 
+# Create docker builder used in creating arm64 images
+# According to the docker documentation, to build the images for ARM64 arch CPU 
+# we would need to use buildx command which uses builder https://docs.docker.com/desktop/multi-arch/
```

Comment:
Looking at https://docs.docker.com/desktop/multi-arch/#build-and-run-multi-architecture-images - I think we need to be clear on (and I am also wondering why it doesn't work 🤷🏻 that's so weird), why the default builder is not sufficient in this scenario?

Is there ongoing work to fix this, that we should keep an eye on?

Reply:
>I think we need to be clear on (and I am also wondering why it doesn't work 🤷🏻 that's so weird), why the default builder is not sufficient in this scenario?

Some clarifications I have got while going through this discussion about multi-arch images, One way could be to change the current approach of using `docker buildx` command and build the images for one platform at a time. Please let me know your thoughts.
Thanks🙂